lib/erlef/blog/post.ex

```diff
@@ -0,0 +1,37 @@
+defmodule Erlef.Blog.Post do
+  @moduledoc """
+  Schema for blog posts.
+  """
+  use Ecto.Schema
```

Comment:
To note you can make use of `use Erlef.Schema` here which will : 

- use Ecto.Schema
- import Ecto.Changeset
- import Erlef.Schema itself for helpers 
- set primary key options (uuid) 
- set timestamp opts 

I'm not normally a huge fan of uuids as a default, but that's what we're using across the board and the data is so small that running into performance issues isn't a concern, to note.